content/dev/dev-guide.md

```diff
@@ -0,0 +1,325 @@
+- Paste the following into the editor and save it.
+  - ```html
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>Hello World</title>
+        <style>body { text-align: center; }</style>
+      </head>
+      <body>
+        <h1>Hello World!</h1>
+        <img src="img/pot.svg" alt="pot" width="219" height="196" />
+      </body>
+    </html>
+    ```
+- **Note**: Why are we using globs to hold separate files instead of copying everything into Clay?
+  - We do this as a current loom limits work around.
+  - Large minimized js blobs in Clay can't be reduced because each one is one line and unique.
+  - This means Clay will grow in size with every javascript file permenantly.
+  - The decision was made to have the js remembered by the userspace application instead of writing to clay.
+  - This prevents Clay from growing permenantly in size.
+  - In the future this will be resolved in two ways: hoon front-end development and Clay tombstones (so we can reduce the size of older js blobs we no longer need).
+  - For the purposes of this guide you don't need to worry about this, it's just helpful context for advanced users.
+
+##### Upload to Glob
+
+We can now create a glob from the directory. To do so, navigate to http://localhost:80/docket/upload in the browser. This will bring up the `%docket` app's Globulator tool.
+
+- Simply select the `hello` desk from the drop-down, click `Choose file` and select the `hello-glob` folder in the the file browser, then hit `glob!`.
+- Now if we return to our ship's homescreen, we should see the tile looks as we specified in the docket file: http://localhost:80
+- And if we click on the tile, it'll load the index.html in our glob.
+- Our app is working!
+
+##### Publish
+
+The final step is publishing our desk with the `%treaty` agent so others can install it. To do this, there's a simple command in the dojo.
+ - ```bash
+   :treaty|publish %hello
+   ```
+   - **Note**: For desks without a docket file (and therefore without a tile and glob), treaty can't be used. Instead you can make the desk public with `|public %desk-name`.
+   - **Note**: Desks aren't publicly readable by default, `:treaty|publish` publishes the metadata and also makes the desk publicly readable, like `|public`.
+
+##### Remote Install
+
+Let's spin up another fake ship so we can try to install our newly published app.
+
+ - ```bash
+   cd ~/urbit-dev
+   ./urbit -F bus
+   ```
+   - Note: For desks without a docket file (and therefore without a tile and glob), users cannot install them through the web interface. Instead remote users can install it from the dojo with `|install ~our-ship %desk-name`.
+- In the browser, navigate to http://localhost:8081 and login with `~bus`'s code which can be found by running `+code` in the dojo for `bus`. In this case, the default is `riddec-bicrym-ridlev-pocsef`. Next, type `~zod/` in the search bar, and it should pop up a list of `~zod`'s published apps, which in this case is our `Hello` app.
```

Comment:
```suggestion
- In the browser, navigate to http://localhost:8081 and login with ~bus's code which can be found by running `+code` in its dojo. For fake ~busses, the code is always `riddec-bicrym-ridlev-pocsef`. Next, type `~zod/` in the search bar, and it should pop up a list of ~zod's published apps, which in this case is our `Hello` app.
```

None of the rest of the document `monospaces` the ship names, I don't think.

Same change below, and in the tips & tricks section.

Reply:
Neither here nor there for this doc, but my preference is to monospace ship names if the style doesn't change the background color, so they still read smoothly.

Reply:
Looked like half and half, defaulted to monospacing them for now.